src/utils/entity-importers/index.js

```diff
@@ -0,0 +1,87 @@
+import Entities from 'constants/entities';
+
+import { has, pick, pickBy, mapValues } from 'constants/lodash';
+import { createId } from 'utils/common';
+
+const createMultipleImporter = (entitiyType, importFunction) => params => {
+  const { data, oldParent, parentEntity } = params;
+  if (!oldParent || !parentEntity) {
+    throw new Error(`Parent must be defined to import multiple ${entitiyType}`);
+  }
+  if (!has(data, entitiyType)) {
+    throw new Error(
+      `The import data has not right structure to import multiple ${entitiyType}`,
+    );
+  }
+
+  const keyMpping = Object.keys(
+    pickBy(
+      data[entitiyType],
```

Comment:
`entityType`

Reply:
Fixed in Irame/sectors-without-number@68b409567571846dd6bbfc0c31fd0ed6a5bafcb3